reviews/views.py

```diff
@@ -1,17 +1,95 @@
 import json
+from django.core.exceptions import MultipleObjectsReturned
 
-from django.http    import JsonResponse
-from django.views   import View
+from django.http  import JsonResponse
+from json.decoder import JSONDecodeError
+from django.views import View
 
 from core.utils      import login_decorator
 from reviews.models  import Review
 from products.models import Product
 
-class ReviewsView(View):
-    def get(self, request):
-        result = [
-            {
-                'user_id' : reviews.user_id,
+class ReviewView(View):
+    @login_decorator
+    def post(self, request): 
+        try:
+            data    = json.loads(request.body)
+            product_id = data['product_id'],
+            content    = data['content'], 
+            image      = data['image'],
+            rating     = data['rating'],
 
+            if not Product.objects.filter(id = product_id).exists():
+                return JsonResponse({'message' : 'DOES_NOT_EXISTS'}, status=404) 
+
+            Review.objects.create(
+                user_id    = request.user.id,
+                product_id = product_id,
+                content    = content,
+                image      = image,
+                rating     = rating
+            )
+            return JsonResponse({'message' : 'SUCCESS'}, status=201)
+       
+        except KeyError:
+            return JsonResponse({'message': 'KEY_ERROR'}, status=400)
+
+        except JSONDecodeError:
+            return JsonResponse({'message':'JSON_DECODE_ERROR'}, status = 400)
+
+    @login_decorator
+    def patch(self, request, review_id):
+        try:
+            data = json.loads(request.body)
+            user = request.user
+
+            if not Review.objects.filter(id=review_id, user_id=user).exists():
+                return JsonResponse({'message' : 'DOES_NOT_EXISTS'}, status=404)
+
+            review = Review.objects.get(id=review_id, user_id=user)
+            
+            review.image    = data['image']
+            review.rating   = data['rating']
+            review.content  = data['content']
+            review.save()
+
+            return JsonResponse({'message' : 'SUCCESS'}, status = 201)
```

Comment:
```suggestion
            return JsonResponse({'message' : 'SUCCESS'}, status = 200)
```
201 에러 코드는 생성 할 때만 사용하면 좋을 것 같아요~